module/rnaseq_de_student-guide.qmd

```diff
@@ -0,0 +1,257 @@
+```
+
+
+If you continue scrolling down the page, you can also examine the pathways and processes the gene product is involved in under the “gene ontology” section. Clicking on the blue squares takes you to a page with more information about how that particular gene was assigned to a pathway or molecular process.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "If you continue scrolling down the page, you can also examine the pathways and processes the gene product is involved in under the “gene ontology” section.", out.width = '100%'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1vikcyml6uw-_DaC1yFpS-UWoWNEGxFukMB_eCi42BqY/edit?slide=id.g346083d4dc0_0_11#slide=id.g346083d4dc0_0_11")
+```
+
+
+Directly underneath the “ontology” section is information about when the gene is expressed during development. You can learn more information by clicking on the blue squares, or by clicking on the links in the upper right-hand corner. These links will take you to other websites.
+
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Directly underneath the “ontology” section is information about when the gene is expressed during development. You can learn more information by clicking on the blue squares, or by clicking on the links in the upper right-hand corner. These links will take you to other websites.", out.width = '100%'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1vikcyml6uw-_DaC1yFpS-UWoWNEGxFukMB_eCi42BqY/edit?slide=id.g346083d4dc0_0_18#slide=id.g346083d4dc0_0_18")
+```
+
+
+# Activity
+A separate Rmd file has been created for this activity. You can download it [here](). Once it’s downloaded, you should open it using RStudio.
+
+## Choose your dataset
+
+You'll first want to decide which dataset(s) you'll use for your exploration. 
+
+### Gut-brain axis: Autism Spectrum Disorder in mice
+
+[Full gene expression data; no comparisons, just counts](https://genomicseducation.org/data/mouse_autism_counts.csv)
+
+**Comparing gene expression between ASD and control mice**
+
+[Both brain regions](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol.csv)
+
+[Prefrontal cortex only](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol_in_prefrontalcortex.csv)
+
+[Striatum only](https://genomicseducation.org/data/mouse_autism_deseq_autismVcontrol_in_striatum.csv)
+
+**Comparing gene expression between prefrontal cortex and striatum**
+
+[All mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype.csv)
+
+[Only ASD mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype_in_ASDmice.csv)
+
+[Only control mice](https://genomicseducation.org/data/mouse_autism_deseq_tissuetype_in_controlmice.csv)
+
+
+## Analysis
+
+Before starting this activity, you should have completed the miniCURE activities and identified genes of interest for your individual project.
```

Comment:
I think I'm still confused how this and the flybase CURE fit together. I think we need to make that super clear.